filesystem/ext2.ksy

```diff
@@ -11,7 +11,7 @@ meta:
   license: CC0-1.0
   endian: le
 instances:
-  # http://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/tree/Documentation/filesystems/ext2.txt#n106
+  doc-ref: http://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/tree/Documentation/filesystems/ext2.txt#n106
```

Comment:
Also invalid (the same problem as [`filesystem/ext2.ksy:138-148`](https://github.com/kaitai-io/kaitai_struct_formats/blob/57f08e40704deddd3d854d61ef3d31310875fc2d/filesystem/ext2.ksy#L138-L148) and [`filesystem/ext2.ksy:264-274`](https://github.com/kaitai-io/kaitai_struct_formats/blob/57f08e40704deddd3d854d61ef3d31310875fc2d/filesystem/ext2.ksy#L264-L274)). You can't put `doc-ref` directly under `instances`.

This won't even compile - try this snippet in the [Web IDE](https://ide.kaitai.io/):

```ksy
meta:
  id: test
instances:
  doc-ref: https://kaitai.io/
  a:
    pos: 0
    type: u1
```

You'll get

```
io.kaitai.struct.format.YAMLParseException: /instances/doc-ref: invalid instance ID: 'doc-ref', expected /^[a-z][a-z0-9_]*$/
```

...since `instances` is merely a `identifier->declaration` dictionary (or _map_). There are no identifiers with special meaning here.